include/bluetooth/services/latency.rst

```diff
@@ -0,0 +1,38 @@
+.. _latency_readme:
+
+GATT Latency Service
+####################
+
+The GATT Latency Service is a custom service that receives data and returns result about GATT latency operations.
+
+To measure the GATT transmission latency, when a client writes a timestamp to the latency characteristic on the server, the server will automatically reply back.
```

Comment:
I believe we can remove the paragraph describing how to measure the latency, as this is done in the application, not in the service itself